src/BookReader/ModeThumb.js

```diff
@@ -235,17 +235,14 @@ export class ModeThumb {
   }
 
   /**
-   * @param {1 | -1} direction
+   * @param {'in' | 'out'} direction
```

Comment:
Q: this arg change is standardizing this function with the other zoom functions ya?

Reply:
ya